PROVISIONING-METAL.md

```diff
@@ -107,7 +106,8 @@ Please keep in mind that when using static addresses, DNS information must be su
   * `via` (IP address): Gateway IP address.  If no gateway is provided, a scope of `link` is assumed.
   * `route-metric` (integer): Relative route priority.
 
-Version `3` adds in support for bonding and vlan tagging.
+Version `3` adds the ability to use a MAC address (colon or dash separated) as the identifier for an interface; this support is limited to interface configuration *only* and may not be used in conjunction with bonds or vlans.
+It also adds support for bonding and vlan tagging.
```

Comment:
nit: You refer to bonds and vlans for the first time in a negative light. Ideally we would introduce bonds and vlans and MAC address features, then tell people about them about any specific problems they might have with these features. I don't know if its better to flip it or not, but possibly we would do: 
```suggestion
Version `3` adds support for bonding, vlan tagging, and the ability to use a MAC address (colon or dash separated) as the identifier for an interface.
MAC address identification is limited to interface configuration *only* and may not be used in conjunction with bonds or vlans.
```
I'm fine being overruled too.